client/me/purchases/remove-purchase/index.jsx

```diff
@@ -117,32 +228,100 @@ const RemovePurchase = React.createClass( {
 				className="remove-purchase__dialog"
 				isVisible={ this.state.isDialogVisible }
 				onClose={ this.closeDialog }>
-				<h1>{ this.translate( 'Remove %(productName)s', { args: { productName } } ) }</h1>
-				{ this.renderDialogText() }
+				<FormSectionHeading>{ this.translate( 'Remove %(productName)s', { args: { productName } } ) }</FormSectionHeading>
+				{ this.renderDomainDialogText() }
 			</Dialog>
 		);
 	},
 
-	renderDialogText() {
+	renderDomainDialogText() {
 		const purchase = getPurchase( this.props ),
 			productName = getName( purchase );
 
-		if ( isDomainRegistration( purchase ) ) {
-			return (
-				<p>
-					{
-						this.translate( 'This will remove %(domain)s from your account.', {
-							args: { domain: productName }
-						} )
-					}
-					{ ' ' }
-					{ this.translate( 'By removing, you are canceling the domain registration. This may stop you from using it again, even with another service.' ) }
-				</p>
-			);
+		return (
+			<p>
+				{
+					this.translate( 'This will remove %(domain)s from your account.', {
```

Comment:
I agree. Wasn't sure if it was my domain to update text that I wasn't already touching.